doc/cprover-manual/cbmc-unwinding.md

```diff
@@ -90,13 +90,14 @@ first use:
 to obtain a list of all loops in the program. Then identify the loops
 you need to set a separate bound for, and note their loop ID. Then use:
 
-    --unwindset L:B,L:B,...
+    --unwindset L:B
 
-where `L` denotes a loop ID and `B` denotes the bound for that loop.
+where `L` denotes a loop ID and `B` denotes the bound for that loop. The
```

Comment:
Description of thread_nr missing.

Reply:
I would say that should go (in a separate PR) into a concurrency-related section.